ui/modules/things/wotDescription.js

```diff
@@ -0,0 +1,84 @@
+/*
+* Copyright (c) 2022 Contributors to the Eclipse Foundation
+*
+* See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+*
+* SPDX-License-Identifier: EPL-2.0
+*/
+
+/* eslint-disable require-jsdoc */
+import * as API from '../api.js';
+import * as Utils from '../utils.js';
+import * as Things from './things.js';
+
+export function WoTDescription(targetTab, forFeature) {
+  let tabLink;
+  let aceWoTDescription;
+  let viewDirty = false;
+  const _forFeature = forFeature;
+  const domTheFeatureId = document.getElementById('theFeatureId');
+
+  const ready = async () => {
+    const tabId = Utils.addTab(
+        document.getElementById(targetTab.itemsId),
+        document.getElementById(targetTab.contentId),
+        'Description',
```

Comment:
I would rather name the tab either:
* WoT TD
* WoT Thing Description
* WoT Description

because it is no "Ditto term".
We could also provide more text as tooltip on the tab I suppose.

Reply:
Decided for "WoT TD" because it takes less space and is similar to "CRUD".
Feel free to change the tooltip.
(would be good to later rework all tooltips with bootstrap tooltips that would allow links and formatting)